source/MRCommonPlugins/Voxels/MRBinaryOperationsPlugin.cpp

```diff
@@ -61,15 +61,41 @@ void BinaryOperations::drawDialog(float menuScaling, ImGuiContext*)
 
     UI::separator(menuScaling, "Operations");
 
-    for (int i = 0; i < int(Operation::Count); ++i)
+    if ( UI::checkbox( "Enable preview", &previewMode_ ) )
     {
-        if (UI::button(operationNames[i].c_str(), { -1, 0 }))
+        if ( previewMode_ )
         {
-            doOperation_(Operation(i));
+            obj1_->setGlobalAlpha( 175 );
+            obj2_->setGlobalAlpha( 175 );
```

Comment:
using global alpha may not look good in browser and on macOS (there is no OpenGL 4.3 that is required for this good transparency rendering), consider using `MeshVisualizeProperty::OnlyOddFragmenst`